creator-node/src/services/sync/secondarySyncFromPrimary.js

```diff
@@ -145,9 +142,12 @@ module.exports = async function (
       // Since different nodes may assign different cnodeUserUUIDs to a given walletPublicKey,
       // retrieve local cnodeUserUUID from fetched walletPublicKey and delete all associated data.
       if (!fetchedCNodeUser.hasOwnProperty('walletPublicKey')) {
-        throw new Error(
-          `Malformed response received from ${creatorNodeEndpoint}. "walletPublicKey" property not found on CNodeUser in response object`
-        )
+        return {
+          error: new Error(
+            `Malformed response received from ${creatorNodeEndpoint}. "walletPublicKey" property not found on CNodeUser in response object`
+          ),
+          result: 'failure_export_wallet'
```

Comment:
these strings are not representative of the error cases - can you either make them specific to each error or consolidate to `failure_malformed_export` or smth